queries/ipkg/highlights.scm

```diff
@@ -0,0 +1,46 @@
+; Keywords
+[
+  "package"
```

Comment:
"package" should be `@keyword.import`

Reply:
but "package" is a name of a currently created package

the `@keyword.import` is more like for `depends = package1, package2`

or `modules = Module1, Module2`

---------------

actually, forget, You are right, the color should be different